pages/people-associated-residence.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div>
+    <div id="main-content" class="pl-5 pr-5 pt-3 pb-3">
+      <div class="flex content-start items-center">
+        <h1 class="text-sm md:text-xl lg:text-2xl">People Associated with the Witch Trials: Residence and Occupation</h1>
```

Comment:
Can remove the styles from this h1 now. Thanks Ruby. Could you have a wee check through the repo to see if there are any other instances you can remove? Thanks!

Reply:
done, have done throughout the full repo. Now the heading looks quite large in some places can you see what you think?
![Uploading image.png…]()

Reply:
I see what you mean. Maybe move the scale for the headings down a bit? h1 at 2.5 rem looks better. It is currently at 3

Reply:
done, have changed in the tailwind config

Reply:
Sorry Ruby I meant change the entire scale. This change means that there is only a .25 rem diff between 5xl and 4xl text. I've tried to look online where the h1, h2 etc sizes correspond to the 2xl etc sizes tailwind sets but can't quite find it.

One thing I did think is that we could just install the Tailwind typography plugin which I think provides a basic font default. Might be easier than us spending ages trying to configure something ourselves...

So I suppose for this issue we need to either:

* shift the whole heading scales down, not just the h1 applied
* _or_ install the typography plugin for tailwind and see if this makes things look better without us needing to tweak things further. 

Cheers!

Reply:
https://baseline.is/tools/type-scale-generator/ - have changed to use the scale from this , using the 1.125 Second Major one. Have also changed so that the are responsive as the headings were taking up far too much space on phone screens before.

I tried using the tailwind typography thing but was more work to configure as had to reconfigure most things.

Ive also made all the info icons inline, it already looked like they were but on phones the were going to the left top middle when title text was going across multiple lines.

Reply:
Looks great. Thanks Ruby